corgidrp/sorting.py

```diff
@@ -0,0 +1,232 @@
+import copy
+import numpy as np
+
+import corgidrp.data as data
+
+def extract_frame_id(filename):
+    """
+    Extract frame ID from an L1 filename. Structure is assumed to be ending
+    like '..._frame_id.fits' where frame_id is a series of digits
+
+    Args:
+      filename: L1 filename
+
+    Returns:
+      Frame id as a string of length 10
+    """
+    idx_0 = len(filename) - filename[::-1].find('_')
+    idx_1 = len(filename) - filename[::-1].find('.') - 1
+
+    return filename[idx_0:idx_1]
+
+def sorting(
```

Comment:
this name should be a little less generic (e.g., `sort_pupilimg_frames()`). In the future, we may have multipel sorting functions here for different data types.

Reply:
I asked you and I used you suggested name: ``sorting.py``. Mine was ``cal_soring``. I found it too generic too. I'll change to this one.